lib/src/kbasesearchengine/main/SearchMethods.java

```diff
@@ -267,6 +268,12 @@ public SearchTypesOutput searchTypes(SearchTypesInput params, String user) throw
     public SearchObjectsOutput searchObjects(SearchObjectsInput params, String user) 
             throws Exception {
         long t1 = System.currentTimeMillis();
+
```

Comment:
We really need to write tests for SearchMethods at some point, but that's not blocking this PR IMO

Reply:
Yep. I'm thinking of writing tests that don't just test or correctness but for quality of results returned.